### PR TITLE
ADD DETAILED INSTRUCTIONS OF SETTING UP EVALAI ON TRAVIS CI FOR DEV & PROD

### PR DESCRIPTION
**Pull request by nsjcorps**
I have added instructions to setup EvalAI on Travis CI for both development and production

### CODE REVIEW

File: DETAILS_TO_SETUP_EVALAI_FOR_DEV_AND_PROD.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+The above will only run if the branch is set to deployment because the deploy command will only run id all conditions in the `on` section are met.
+
+### It must not be only branches other conditions could be used. Other conditions could be used in the `on:` section.
+
+Example
+
+```
+deploy:
+  provider: s3
+  acces_key_id = "Key ID"
```
Comment (yashdusing) [inline DETAILS_TO_SETUP_EVALAI_FOR_DEV_AND_PROD.md]: @nsjcorps Could you change this to access

File: DETAILS_TO_SETUP_EVALAI_FOR_DEV_AND_PROD.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  secret_access_key = "Secret"
+  bucket: "S3 Bucket"
+  on:
+    condition: "$CC = gcc"
+ ```
+ 
+ The above will only run is the env variable `$CC` is set to gcc.
```
Comment (yashdusing) [inline DETAILS_TO_SETUP_EVALAI_FOR_DEV_AND_PROD.md]: Also, change the first 'is' to 'if'

**Comment by nsjcorps** [comment]
@yashdusing Done!!

**Comment by nsjcorps** [comment]
@yashdusing Done!!

**Comment by yashdusing** [comment]
LGTM 👍

**Comment by nsjcorps** [comment]
Can I have the task accepted? @yashdusing

**Comment by yashdusing** [comment]
@nsjcorps I approved the task

**Comment by yashdusing** [comment]
@nsjcorps I have approved the task

**Comment by nsjcorps** [comment]
Thank you. @yashdusing

**Pull request closed without merging**